modules/nf-core/ensemblvep/vep/main.nf

```diff
@@ -33,7 +33,6 @@ process ENSEMBLVEP_VEP {
     def prefix = task.ext.prefix ?: "${meta.id}"
     def dir_cache = cache ? "\${PWD}/${cache}" : "/.vep"
     def reference = fasta ? "--fasta $fasta" : ""
-
     """
     vep \\
         -i $vcf \\
```

Comment:
For the flags to use with vep:
1) we could consider `--config` and put everything in a seperate file. 
2) in the config I would suggest including: 
```
species
assembly
stats_file	
fork	
cache_version
cache 
custom
vcf
overlaps
regulatory
check_existing
check_svs
database
```
with the custom flag we can add the CRE, TAD, RetNet, RetNetCNV and IRD bed files (as a `--debaere`option) and add the option for the user to include a path to a custom bed file.

Reply:
I wouldn't do it like that. Nextflow uses its own configs which are more dynamic than VEP configs. You can change these values using a config you give to the pipeline. 

Would that BED file be sample specific or a common BED file for all samples?